### ISSUE TITLE
Search endpoint parameters wrong / incomplete

### ISSUE THREAD

**Issue by homersimpsons** [elastic/elasticsearch-php#1070]
### Summary of problem or feature request

Search endpoint parameters seems wrong / incompletes

They are defined there:
https://github.com/elastic/elasticsearch-php/blob/c03a02cdd8fb05e5bcab90c9a68e3a0b1af59073/src/Elasticsearch/Endpoints/Search.php#L50-L94

And also in the PHPDoc here:
https://github.com/elastic/elasticsearch-php/blob/c03a02cdd8fb05e5bcab90c9a68e3a0b1af59073/src/Elasticsearch/Client.php#L1155-L1199

To me thoses should reflect the one described in https://www.elastic.co/guide/en/elasticsearch/reference/current/search-search.html

For example `analyzer` is in the list of this repo but is not in the linked documentation. Every item in the documentation seems to be present in this repository though.

Other endpoints might have the same issue (I did not check).

**Comment by ezimuel** [comment]
@homersimpsons All the endpoints are generated using the Elasticsearch API specification. For instance, these are the specs for Elasticsearch ver. 7.9.2:
- [Oss endpoints](https://github.com/elastic/elasticsearch/tree/v7.9.2/rest-api-spec/src/main/resources/rest-api-spec/api)
- [Xpack endpoints](https://github.com/elastic/elasticsearch/tree/v7.9.2/x-pack/plugin/src/test/resources/rest-api-spec/api)

Regarding the [Search](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-search.html) API the parameters are reported in this spec file: https://github.com/elastic/elasticsearch/blob/v7.9.2/rest-api-spec/src/main/resources/rest-api-spec/api/search.json#L50-L246

Since the **Search API** contains a long list of parameters, the documentation is splitted in different pages. For instance, the `analyzer` parameter is reported here: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-explain.html#sample-api-query-params

Long story short, the most updated list of API parameters is reported in the [Client](https://github.com/elastic/elasticsearch-php/blob/v7.9.1/src/Elasticsearch/Client.php) class and in all the [Namespaces](https://github.com/elastic/elasticsearch-php/tree/v7.9.1/src/Elasticsearch/Namespaces) of elasticsearch-php.
